config/overlays/mutation_webhook/webhook-config.yaml

```diff
@@ -0,0 +1,28 @@
+# TODO: this is a temporary placement for the mutation webhook
+# It is kept separate until the mutation feature is stable enough.
+# Then the mutation webhook should be moved to config/webhooks/manifests.yaml  
+
+apiVersion: admissionregistration.k8s.io/v1beta1
```

Comment:
This file looks exactly like config/overlays/mutation_webhook/webhook.yaml. Whats the reason for renaming this file?